### PR TITLE
v1.13.1-beta.1

### PR DESCRIPTION
**Pull request by artonge**

### CODE REVIEW

File: lib/Capabilities.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function getCapabilities(): array {
 		$capabilities = ['end-to-end-encryption' =>
 			[
 				'enabled' => true,
-				'api-version' => '2',
+				'api-version' => '1.2',
```
Comment (nickvergessen) [inline lib/Capabilities.php:54]: https://github.com/nextcloud/end_to_end_encryption/blob/master/tests/Unit/CapabilitiesTest.php#L78

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+* [master] Update nextcloud/ocp dependency by @nextcloud-command in https://github.com/nextcloud/end_to_end_encryption/pull/377
+* Remove parent fetching optimisation by @artonge in https://github.com/nextcloud/end_to_end_encryption/pull/430
+* Adapt filedrop to v1.2 by @artonge in https://github.com/nextcloud/end_to_end_encryption/pull/435
+* [master] Update nextcloud/ocp dependency by @nextcloud-command in https://github.com/nextcloud/end_to_end_encryption/pull/436
+* Chore(deps): Bump webpack from 5.75.0 to 5.76.1 by @dependabot in https://github.com/nextcloud/end_to_end_encryption/pull/403
+
+## New Contributors
+* @kesselb made their first contribution in https://github.com/nextcloud/end_to_end_encryption/pull/400
+
+**Full Changelog**: https://github.com/nextcloud/end_to_end_encryption/compare/v1.12.4...1.13.1
```
Comment (nickvergessen) [inline CHANGELOG.md:21]: Should be with v?
```suggestion
**Full Changelog**: https://github.com/nextcloud/end_to_end_encryption/compare/v1.12.4...v1.13.1
```

**Review by nickvergessen — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (artonge) [reply CHANGELOG.md]: I also added the beta.1 suffix

**Comment by nickvergessen** [comment]
Force merging as per app maintainer request

**Pull request merged**